includes/Database_Upgrader.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Class Database_Upgrader
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class Database_Upgrader
+ *
+ * @package Google\Web_Stories
+ */
+class Database_Upgrader {
+
+	/**
+	 * The slug of database option.
+	 *
+	 * @var string
+	 */
+	const OPTION = 'web_stories_db_version';
+
+	/**
+	 * The slug of database option.
+	 *
+	 * @var string
+	 */
+	const PREVIOUS_OPTION = 'web_stories_previous_db_version';
```

Comment:
What's a use case where we'd need the previous db version?

Reply:
There is no current use for it. It might be useful in the future.

Reply:
There maybe migration where you move from 1 -> 9 where the steps are different. I know we don't have a use this now, but if we don't store this, then we will not have this data.